README.md

```diff
@@ -1,56 +1,48 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Leaderboard
 
-# Project Name
-
-> Description the project.
+> A leaderboard project made with webpack that fetches data from an API.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
+- Git
+- Webpack
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] (OPTIONAL) Kindly remove fix this live demo link or remove it completely as it is broken in order to make your README file more professional 👍

Reply:
Will do. Thanks for the feedback 👍